app/src/organisms/ModuleWizardFlows/SelectLocation.tsx

```
@@ -69,7 +70,11 @@ export const SelectLocation = (
   const { t } = useTranslation('module_wizard_flows')
   const moduleName = getModuleDisplayName(attachedModule.moduleModel)
   const handleOnClick = (): void => {
-    createMaintenanceRun({})
+    if (maintenanceRunId === null) {
+      createMaintenanceRun({}).catch(error => {
+        console.error('Could not create maintenance run', error.message)
```

Instead of logging this error, let's error out the flow if this happens. You should be able to do `props.setShowErrorMessage(error.message)`